mypy/nodes.py

```diff
@@ -96,6 +96,16 @@ def get_column(self) -> int: pass
     'typing.Deque': '__mypy_collections__.deque',
 }
 
+reverse_collection_aliases = dict((name.replace('__mypy_collections__', 'collections'), alias)
+                                  for alias, name in
+                                  collections_type_aliases.items())  # type: Dict[str, str]
+
+nongen_builtins = {'builtins.tuple': 'typing.Tuple',
+                   'builtins.frozenset': 'typing.FrozenSet',
```

Comment:
I'm curious why this isn't in `type_aliases` above. How is frozenset different from set?